### DISCUSSION TITLE
Optimizing Netdata Disk Charts: Excluding "Reserved for Root" Sections

### DISCUSSION THREAD

**Discussion by ghost** [netdata/netdata#17558] [Q&A]
**Optimizing Netdata Disk Charts: Excluding "Reserved for Root" Sections**

I've configured the Netdata Disk Statistics plugin to monitor disk performance metrics on my system. While the charts provide valuable insights into disk usage and performance. 
![image](https://github.com/netdata/netdata/assets/87641643/1e5a427f-0ccf-40d9-920b-29a7c34cc2b7)

I'd like to exclude / remove **reserved for root** from the graphs ( /var/lib/docker )
![2024_04_30_0y9_Kleki](https://github.com/netdata/netdata/assets/87641643/18f96be2-e48c-479a-94f3-702a5c56b504)

![2024_04_30_0y6_Kleki](https://github.com/netdata/netdata/assets/87641643/d9c2484e-d0fc-4747-a154-210609918bba)

Here's a summary of my current Netdata configuration:

```ini
[plugin:proc:diskspace]
    update every = 60
    exclude space metrics on paths = /proc/* /sys/* /var/run/user/* /run/user/* /var/lib/docker/* /dev /dev/shm /run /run/* /opt/scorer/lib/stack-mgr/stacks/custom
    inodes usage for all disks = no

[plugin:proc:/proc/diskstats]
    exclude disks = loop* ram* boot rootfs rex-kmshare ext4
    merged operations for all disks = no
    backlog for all disks = no
    i/o time for all disks = no
    enable new disks detected at runtime = no

[plugin:proc:/proc/diskstats:sda]
    enable = yes
    enable performance metrics = auto
    bandwidth = auto
    operations = auto
    merged operations = no
    queued operations = auto
    utilization percentage = auto
    extended operations = no

[plugin:proc:/proc/diskstats:nvme0n1]
    enable = yes
    enable performance metrics = auto
    bandwidth = auto
    operations = auto
    merged operations = no
    queued operations = auto
    utilization percentage = auto
    extended operations = no
```
As you can see, I've excluded certain paths from disk space metrics to avoid cluttering the charts. However, I'm unsure how to **reserved for root** from the graphs ( /var/lib/docker ) & ( /opt/scorer/log )

Here's the output of `lsblk -f` for reference:

```
root@DEB-NUC11TNH-BTTN22900SU1:/opt/scorer/etc/netdata# lsblk -f
NAME        FSTYPE FSVER LABEL UUID                                 FSAVAIL FSUSE% MOUNTPOINTS
nvme0n1                                                                            
├─nvme0n1p1 vfat   FAT32       71BF-1B83                             505.2M     1% /boot/efi
├─nvme0n1p2 ext4   1.0         2f126303-5a9a-445f-b776-5fded8b26359       0    95% /
├─nvme0n1p3 ext4   1.0         0e4c9658-01af-44d3-85c3-42438e00fef8    5.5G     1% /opt/scorer/log
└─nvme0n1p4 ext4   1.0         04109503-d561-49c2-9107-4a35bb8f6313   99.9G    46% /opt/scorer/lib/stack-mgr/stacks/custom
                                                                                   /home/scorer/work
                                                                                   /var/lib/docker
```
Could someone provide guidance on how to exclude **reserved for root** from the graphs ( /var/lib/docker ) & ( /opt/scorer/log ) 
Your help would be greatly appreciated! Thanks in advance.

---

**Comment by ilyam8** [comment]
Hi, it is not possible to exclude (remove) this dimension. If it bothers you - consider unselecting it when viewing the chart.

<img width="1298" alt="Screenshot 2024-05-01 at 11 38 44" src="https://github.com/netdata/netdata/assets/22274335/a77315fe-296f-44b6-b78e-eb9f4ab80d0c">

---

I see you are using the old dashboard (UI) - it is not longer supported and won't be updated.
